src/test/java/test/listeners/github1284/Test1284.java

```diff
@@ -0,0 +1,11 @@
+package test.listeners.github1284;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Test1284 {
+    @Test
+    public void test1284Method() {
+        Assert.assertNotNull(Listener1284.getInstance());
```

Comment:
The test only checks if the listener is created but not invoked, and not when it is invoked.
Could you check the order of invoked methods instead?